test/test_utils/__init__.py

```diff
@@ -876,50 +883,57 @@ def parse_canary_images(framework, region):
     framework_versions = versions if len(versions) < 4 else versions[:3]
     dlc_images = []
     for fw_version in framework_versions:
-        py3_version = get_canary_default_tag_py3_version(framework, fw_version)
```

Comment:
this is probably the only usage of this function, so you can prob remove the function too

Reply:
Yes, this is the only occurrence - removed